modules/fedimint-wallet/src/lib.rs

```diff
@@ -970,6 +970,16 @@ impl Wallet {
             ));
         }
 
+        match bitcoind_rpc.get_block_height().await {
+            Ok(height) => info!(height, "Connected to bitcoind"),
+            Err(err) => warn!("Bitcoin node is not ready or configured properly. Modules relying on it may not function correctly: {:?}", err),
+        }
+
+        match bitcoind_rpc.get_fee_rate(1).await {
+            Ok(feerate) => info!(feerate = feerate.ok_or(0), "Bitcoind feerate available"),
```

Comment:
```
 fedimint-workspace-build> 979 |             Ok(feerate) => info!(feerate = feerate.ok_or(0), "Bitcoind feerate available"),
```

Weird. Maybe `feerate = ( ... )` is needed etc. Or just `match` again or something.

Reply:
It was complaining that tracing::Value trait is not implemented by Feerate. I couldn't figure out how to do that. Since there is only one property in Feerate struct, I thought we can output that in the logs for now, and then revisit how to enable tracing by implementing the relevant tracing traits, if that is the right way to go about it.